spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -101,28 +101,21 @@ def __init__(self, parent, id_,
                  options_button=None,
                  show_elapsed_time=False,
                  reset_warning=True,
-                 ask_before_restart=True,
-                 ask_before_closing=False,
```

Comment:
ask_before_restart and ask_before_closing are not really related to ClientWidget and more related to the main_widget